Sources/EntitiesMP/Common/Particles.cpp

```diff
@@ -1591,7 +1591,7 @@ INDEX Particles_Regeneration(CEntity *pen, FLOAT tmStart, FLOAT tmStop, FLOAT fY
     
     UBYTE ubR = (UBYTE) (192+afStarsPositions[iRnd][1]*64);
     UBYTE ubG = (UBYTE) (192+afStarsPositions[iRnd][2]*64);
-    UBYTE ubB = (UBYTE) (192+afStarsPositions[iRnd][3]*64);
+    UBYTE ubB = (UBYTE) (192+afStarsPositions[iRnd][3]*64); // FIXME DG: afStarsPositions[i][3] invalid, what's going on here?
```

Comment:
ok, now I'm really confused, found that code in the same file:
```c++
    FLOAT fRndAppearX = afStarsPositions[iRnd][0]*vStretch(1);
    FLOAT fRndSpeedY = (afStarsPositions[iRnd][1]+0.5f)*0.125f*vStretch(2);
    FLOAT fRndAppearZ = afStarsPositions[iRnd][2]*vStretch(3);
    FLOAT3D vRndDir=FLOAT3D(afStarsPositions[iRnd][1],0,afStarsPositions[iRnd][3]);
```
there indices 0-3 are all used within 4 lines

OTOH, all those values seem to be completely random (see `InitParticleTables()`) so it probably doesn't matter